### PR TITLE
Django1.9.7support

### PR DESCRIPTION
**Pull request by lsetiawan**
@miguelcleon Here's a cleaner pull request. Thanks.

### CODE REVIEW

File: templatesAndSettings/settings.py [hunk 1]
```diff
@@ -6,27 +6,27 @@
 secret_key = 'random_secret_key_like_so_7472873649836'
 
 
-media_root = 'C:/Users/leonmi/Google Drive/ODM2Djangoadmin/ODM2CZOData/upfiles/'
+media_root = '/Users/lsetiawan/Desktop/shared_ubuntu/APL/ODM2/ODM2-Admin/ODM2CZOData/upfiles/'
```
Comment (miguelcleon) [inline templatesAndSettings/settings.py:9]: let's not push any changes to settings.py unless they are really needed. This file should only contain specifications that are specific to a particular instance of the application.

File: templatesAndSettings/settings.py [hunk 2]
```diff
@@ -6,27 +6,27 @@
...
-        'HOST': 'x', #micro server  '52.20.81.11'
-        'PORT': 'x',
+        'NAME': 'odm2',
+        'USER': 'lsetiawan',
+        'PASSWORD': '',
+        'HOST': 'localhost', #micro server  '52.20.81.11'
+        'PORT': '5432',
      'OPTIONS': {
-      'options': '-c search_path=odm2,odm2extra'
+      'options': '-c search_path=admin,odm2,odm2extra'
```
Comment (miguelcleon) [inline templatesAndSettings/settings.py:29]: addinf this search_path for admin should be the only change we want to apply to this file.

File: templatesAndSettings/settings.py (hunk 1 above)
Comment (miguelcleon) [inline templatesAndSettings/settings.py:9]: I want to try it so I'll pull for now and just push back the changes.

Comment (lsetiawan) [inline templatesAndSettings/settings.py:9]: @miguelcleon my apologies, I will fix this right away. thanks.

**Pull request merged**